src/lib/browse/PCBoundaryLayer.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import {
+    hoveredToggle,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+    overwritePolygonLayer,
+  } from "../../maplibre_helpers";
+  import { map } from "../../stores";
+  import {
+    ColorLegend,
+    ExternalLink,
+    HelpButton,
+    InteractiveLayer,
+  } from "../common";
+  import { Checkbox } from "../govuk";
+  import { colors } from "./colors";
+
+  let name = "parliamentary_constituencies";
+  let color = colors.parliamentary_constituencies;
+  let outlineLayer = `${name}-outline`;
+
+  overwritePmtilesSource(
+    $map,
+    name,
+    `https://atip.uk/layers/v1/${name}.pmtiles`
+  );
+
+  overwritePolygonLayer($map, {
+    id: name,
+    source: name,
+    sourceLayer: name,
+    color,
+    opacity: hoveredToggle(0.5, 0.1),
+  });
+  overwriteLineLayer($map, {
+    id: outlineLayer,
+    source: name,
+    sourceLayer: name,
+    color,
+    width: 5,
+  });
+
+  let show = false;
+  // InteractiveLayer manages the polygon layer, but we also need to control the outline
+  $: {
+    if ($map.getLayer(outlineLayer)) {
+      $map.setLayoutProperty(
+        outlineLayer,
+        "visibility",
+        show ? "visible" : "none"
+      );
+    }
+  }
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    return `<p>${feature.properties.Name} (${feature.properties.Census_Code})</p>`;
+  }
+
+  function onClick(e: CustomEvent<MapGeoJSONFeature>) {
```

Comment:
This might be really unprofessional seeming, but I think a reliable way to surface the information of who's MP (iirc this was something they asked for) would be to open a google (or similar) search in a new tab for `${constituency name} MP`. Not very professional but probably reliable

Reply:
Even better, I found an official site where you can input the PC name and reliably get the MP and contact details out!